common/Generieke-Datatypen-Gemeenten.yaml

```diff
@@ -0,0 +1,79 @@
+openapi: 3.0.0
+info:
+  title: Generieke datatypen voor Gemeentelijke APIs onder regie van VNGRealistaie
+  description: |
+    De definitie van generieke datatypen die worden gebruikt door gemeentelijke API's.
+    Deze datatypen komen overeen met het basismodel Generieke Datatype Gemeenten waarnaar verwezen wordt vanuit informatiemodellen.
+    Er worden uit dit bestand met herbruikbare componenten geen componenten verwijderd.
+    Als een component in de toekomst niet meer hergebruik mag worden dan wordt het deprecated verklaard.
+  version: '1.0.0'
+  contact:
+    name: Generieke Datatypen Gemeenten.
+    url: https://github.com/VNG-Realisatie/API-Kennisbank/Common
+  license:
+    name: European Union Public License, version 1.2 (EUPL-1.2)
+    url: https://eupl.eu/1.2/nl/
+paths: {}
+components:
+  schemas:
+    UUID:
+      type: "string"
+      minLength: 36
+      maxLength: 36
+      example: "123e4567-e89b-12d3-a456-426614174000"
+    OIN:
+      type: "string"
+      minLength: 20
+      maxLength: 20
+      pattern: ^[0-9]{20}$
+      description: |
+                    Het Organisatie-identificatienummer (OIN) is een uniek nummer dat Logius kan toekennen aan organisaties om zich te kunnen identificeren, authentiseren en of autoriseren bij digitaal berichtenverkeer binnen en met de overheid.
+    DatumOnvolledig:
+      type: "object"
+      description: |
+                    Gegevens over de datums die mogelijk niet volledig zijn.
+      properties:
+        dag:
+          type: "integer"
+          description: |
+                        Als de dag van de datum bekend is wordt dit element gevuld, ook als de volledige datum bekend is.
+          minimum: 1
+          maximum: 31
+          example: 3
```

Comment:
Waarom heeft dit attribute in het SIM model het type 'integer'. Ik zou daar het type 'Day' verwachten (ongeacht hoe we het vertalen naar dit schema).

In mijn stylesheets vertaal ik op dit moment het type 'Day' naar 'string', ik zal dat aanpassen zodat het wordt vertaalt naar 'integer'.

Reply:
k heb hier in ieder geval de definitie vanuit Haal Centraal overgenomen. Ik vind Day ook uitstekend, maar op deze manier kan ik wel expliciet minimum en maximum opgeven. Bij Day doe je dat niet en moeten deze dus in jouw script meegegenereerd worden (dus Day genereert niet alleen naar een integer, maar ook een minimum 1 en maximum 31).

Reply:
Waarom kan je geen minimum en maximum opgeven in het model? je kunt toch gewoon zeggen dat het attribute van het type 'Day' is en de tv's voor minimum en maximum definiëren?
Daarnaast hoeft mijn script hier niets mee te doen. Mijn script zorgt er immers voor dat als het datatype 'DatumOnvolledig' wordt gebruikt er wordt verwezen naar het component 'DatumOnvolledig' in dit schema.

Reply:
Ik ben er ook niet principieel tegen hoor, maken we er Day, Month en Year van....

Reply:
Prima. Ik genereer in json trouwens ook nog 

		"format": "date_mday"

En voor year en month:

		"format": "date_fullyear"
		"format": "date_month"

Moet dat er in dit schema ook bijkomen of moet ik dit dus juist niet genereren?

Reply:
Hoewel het dan wel weer vreemd is dat ik in de yaml header wel:

		format: jaar

genereer en voor month en day niets overeenkomstigs.